### ISSUE TITLE
IsNodeOnSetPtManager only checks SPM:Scheduled

### ISSUE THREAD

**Issue by jmarrec** [NREL/EnergyPlus#6069]
Issue overview
--------------
In [SetPointManager::IsNodeOnSetPtManager](https://github.com/NREL/EnergyPlus/blob/a5b04d73ecfef706cda7d0543ccad31e7c27087a/src/EnergyPlus/SetPointManager.cc#L7877)

It only checks for `SchSetPtMgr`, which if you track down, is actually only `SetpointManager:Scheduled`. I find that strange since the docstring of the routine doesn't mention it, and the capacity to check for other SPMs is already implemented, see for example `SetPointManager:OutdoorAirReset ` at [SetPointManager.cc#L1057](https://github.com/NREL/EnergyPlus/blob/a5b04d73ecfef706cda7d0543ccad31e7c27087a/src/EnergyPlus/SetPointManager.cc#L1057)

I think this loop should be done on `AllSetPtMgr` instead of `SchSetPtMgr`, shouldn't it?

### Details

Some additional details for this issue: 
* EnergyPlus version 8.6 (but checked the code in the current develop...)
* Defect example:

This affects in particular the ability to place anything but a `SetpointManager:Scheduled` for the secondary side of a `PlantLoop` having a `TwoWayCommonPipe` option.

Take the `TwoWayCommonPipe_Pri-Sec.idf`. It runs fine. Now change the secondary SPM located [L3261](https://github.com/NREL/EnergyPlus/blob/8df72926681d9827a5ea69ab4303f30df39b1f43/testfiles/TwoWayCommonPipe_Pri-Sec.idf#L3261) from:

```
  SetpointManager:Scheduled,
    Chilled Water Secondary Loop Setpoint Manager,  !- Name
    Temperature,             !- Control Variable
    COMPACT HVAC-ALWAYS 8.00,!- Schedule Name
    Chilled Water Loop Supply inlet;  !- Setpoint Node or NodeList Name
```

to:

```
 SetpointManager:OutdoorAirReset,
    Chilled Water Secondary Loop Setpoint Manager,  !- Name
    Temperature,             !- Control Variable
    11.11,                   !- Setpoint at Outdoor Low Temperature {C}
    7.22,                    !- Outdoor Low Temperature {C}
    7.22,                    !- Setpoint at Outdoor High Temperature {C}
    29.44,                   !- Outdoor High Temperature {C}
    Chilled Water Loop Supply inlet;  !- Setpoint Node or NodeList Name
```

Now you'll get this error:

```
   ** Severe  ** GetPlant/CondenserLoopData: PlantLoop="CHILLED WATER LOOP", Invalid condition.
   **   ~~~   ** While using a two way common pipe there must be a setpoint in addition to the Plant Supply Outlet node.
   **   ~~~   ** Currently neither plant demand inlet nor plant supply inlet have setpoints.
   **   ~~~   ** Select one of the two nodes and rerun the simulation.
   **  Fatal  ** GetPlant/CondenserLoopData: Errors found in processing input. Preceding conditions cause termination.
```

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
Closed via #6142

**Comment by jmarrec** [comment]
Thank you guys.
